README.md

```diff
@@ -217,14 +206,30 @@ Install additional build dependencies:
 sudo apt-get install python-vcstool python3-colcon-common-extensions
 ```
 
-Download and build MoveIt:
+Download and build MoveIt2:
 ```bash
 mkdir -p ~/moveit2_ws/src
 cd ~/moveit2_ws/src
 git clone https://github.com/AcutronicRobotics/moveit2 -b master
 cd ..
 vcs import src < src/moveit2/moveit2.repos
-colcon build --merge-install --cmake-args -DBUILD_TESTING=FALSE
+export ROS_DISTRO=dashing
+rosdep update && rosdep install -q -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO}
+colcon build --merge-install
+```
+
+### Using a Docker container
+
+```bash
+docker run -it ros:dashing
```

Comment:
You might want to explicitly point out that the first command is for the host machine and the remaining commands are meant to be executed inside the container.

Reply:
Thanks for the review! I'll include this!